integration_test/third_party_apps_data/applications/activemq/sles/post

```diff
@@ -0,0 +1,7 @@
+set -e
+
+<<<<<<< HEAD
```

Comment:
Another one.

Reply:
Sorry about that, unsure how that happened as well, should be fixed now

Reply:
Thanks!